app/src/main/java/ch/epfl/sweng/studyup/firebase/Firestore.java

```diff
@@ -204,6 +209,10 @@ public void addQuestion(final Question question) {
         db.collection(FB_QUESTIONS).document(question.getQuestionId()).set(questionData);
     }
 
+    public void updateQuests(Map<String, Object> remotePlayerData) {
```

Comment:
Is it an oversight? Is it normal?

Reply:
I did not put this in here.

Reply:
Strange. I did not write this, not sure how it ended up here.